_summaries/wu2021rethinking_1.md

```diff
@@ -1,19 +1,180 @@
 ---
 layout: summary
-title: Summary
-paper: {{paper_tag}}
+title: Rethinking “Batch” in BatchNorm
+paper: wu2021rethinking_1
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: # chengchunhsu
+score: 8/10
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+**A short review on BatchNorm**
+
+BatchNorm is sometimes viewed as a “necessary evil” in the design of CNNs
+
+- It improves model convergence speed
+
+- regularizing effect to combat overﬁtting
+
+
+
+![BN_EQ1](wu2021rethinking_1_1.jpg)
+
+![BN EQ2](wu2021rethinking_2.jpg)
+
+
+
+Sampling strategies:
+
+* Training: mini-batch samples
+* Inference: whole dataset
+
+
+
+Design issue: how to compute **μ** and **σ^2**?
+
+- size
+- data source
+- algorithm to compute the statistics
+
+
+
+---
+
+**Issue #1: Compute *μ* and *σ^2* during training**
+
+Discussion between two statistics computation methods - EMA and PreciseBN.
+
+
+
+<u>Exponential moving average (EMA)</u>: updated in every training iteration
+
+![EMA equation](wu2021rethinking_3.jpg)
+
+
+
+Problems:
+
+- When λ is large - the statistics outdated
+- When λ is small - ignore the whole population
+
+
+
+<u>PreciseBN</u>:
+
+- apply the (ﬁxed) model on many mini-batches to collect batch statistics
+- aggregate the per-batch statistics into population statistics
+
+
+
+Result:
+
+- PreciseBN is better and also more stable than with EMA
```

Comment:
We probably need to detail which tasks which  preciseBN is better and more stable?